core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -6778,6 +6785,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String WORKER_RPC_EXECUTOR_FJP_ASYNC =
         "alluxio.worker.rpc.executor.fjp.async";
     public static final String WORKER_SESSION_TIMEOUT_MS = "alluxio.worker.session.timeout";
+    public static final String WORKER_STARTUP_TIMEOUT =
+        "alluxio.worker.startup.timeout";
```

Comment:
nit: I guess we don't need a separate line.

Reply:
I'll fix it.